protobufs/livekit_metrics.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package livekit;
+option go_package = "github.com/livekit/protocol/livekit";
+option csharp_namespace = "LiveKit.Proto";
+option ruby_package = "LiveKit::Proto";
+
+
+message MetricsBatch {
+  repeated TimeSeriesMetric time_series = 1;
+  repeated AnnotationMetric annotations = 2;
+}
+
+message TimeSeriesMetric {
+  // Metric name, e.g., "speech_probability"
+  string label = 1;
```

Comment:
> Only specific labels will be recorded on our servers for the Dashboard. (e.g lk-speech-probability, lk-llm-ttft)

Do we want to define these somewhere ?
I understand the sentiment that new labels shouldn't require protocol updates, but for stuff that we record, it might be nice for maintainability across different implementations.

Reply:
It's hard to know what are the labels we will record today (there is a lot..)
I was thinking about doing something similar to how we handle data messages topics (for e.g `lk-chat-topic`).

We could create an enum holding all our internal/records labels. On downside is that protobuf doesn't support string enums, so in the future when we're going to export data to our customers. We will have to provide two different ways of using our labels? (oneof str/our_enum)